.github/workflows/eden_gcp.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: EdenGCP
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: ["publish"]
+    branches: [main]
+    types:
+      - completed
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        hv: ["kvm", "xen"]
+    steps:
+      - name: get eden
+        uses: actions/checkout@v2
+        with:
+          repository: 'lf-edge/eden'
+      - name: fetch or build eve
+        env:
+          TAG: pr${{ github.event.pull_request.number }}
+          CACHE: evebuild/danger
+        run: |
+          if docker pull "$CACHE:$TAG-${{ matrix.hv }}"; then
+             docker tag  "$CACHE:$TAG-${{ matrix.hv }}" "lfedge/eve:$TAG-${{ matrix.hv }}"
+             docker tag  "$CACHE:$TAG-${{ matrix.hv }}" "lfedge/eve:$TAG-${{ matrix.hv }}-amd64"
+          else
+            echo "No tag $CACHE:$TAG-${{ matrix.hv }} found"
+          fi
+          echo "TAG=$TAG" >> $GITHUB_ENV
+      - name: Public IP
+        id: ip
+        uses: actions/http-client
+        result-encoding: string
+        script: |
+          const maxRetries = parseInt(core.getInput('maxRetries'), 10);
+          const http = new HttpClient('haythem/public-ip', undefined, { allowRetries: true, maxRetries: maxRetries });
+          try {
+            const ipv4 = await http.getJson<IPResponse>('https://api.ipify.org?format=json');
+          } catch (error) {
+            echo ("Didn't get anything")
+            echo(error.message)
+          }
+          return ipv4
+      - name: setup packages
+        run: |
+          sudo apt update
+          sudo apt install -y qemu-utils openvpn
+          echo "$OVPN_FILE" | base64 -d > ./config.ovpn
+        env:
+          OVPN_FILE: ${{ secrets.OVPN_FILE }}
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: clean
+        run: |
+          gcloud compute instances delete eve-edengcp-actions-${{ matrix.hv }}-${{github.run_number}} -q --zone=us-west1-a || echo "Does not exist"
+          gcloud compute images delete eve-edengcp-actions-${{ matrix.hv }}-${{github.run_number}} -q || echo "Does not exist"
+      - name: Connect VPN
+        id: connect_vpn
+        timeout-minutes: 1
+        run: |
+          sudo openvpn --config ./config.ovpn --daemon
+          until ip -f inet addr show tun0; do sleep 5; ip a; done
+          echo ::set-output name=tunnel_ip::$(ip -f inet addr show tun0 | sed -En -e 's/.*inet ([0-9.]+).*/\1/p')
+      - name: build eden
+        run: |
+          make build
+          make build-tests
+      - name: setup
+        run: |
```

Comment:
since you'll be dropping 'fetch or build eve' step -- you should add TAG setting env section here and set the tag to